### ISSUE TITLE
Sending more than one mail in same second will overwrite saved eml in testMode

### ISSUE THREAD

**Issue by bkonetzny** [vernes/YiiMailer#40]
We noticed this when debugging bulk mail sendings.

Filename for testMode is determined by this line in the save method:
```php
$filename = date('YmdHis') . (!empty($this->view) ? '_' . $this->view : '') . '_' . uniqid() . '.eml';
```

So mails with identical view and sent in same second will overwrite the eml file.

Fix could be as follows:
```php
$filenameParts = [
  date('YmdHis'),
  uniqid(),
];

if (!empty($this->view)) {
  $filenameParts[] = $this->view;
}

$filename = implode('_', $filenameParts) . '.eml';
```

**Comment by vernes** [comment]
Did you test your fix? I can't see how it is any different, i'ts the same 3 parts. `uniqid()` is there to ensure uniqueness and down to microseconds. Maybe to increase entropy with `uniqid('', true)`?

**Comment by bkonetzny** [comment]
Hey @vernes,
sorry, the issue is with YiiMailer 3.x and the actual code there is as follows:
`$filename = date('YmdHis') . '_' . (empty($this->view) ? uniqid() : $this->view) . '.eml';`

As it seems to be fixed in 4.x (thats where I got the snippet from), I'll try to upgrade to 4.x.

**Comment by vernes** [comment]
I have just published v4.0.1 which includes fix with uniqid. That should solve your problem.
